DemoViper/Sources/Extensions/UIBarButtonItem+Identifier.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIBarButtonItem+Identifier.swift
+//  DemoViper
+//
+//  Created by Nguyen M. Tam on 11/01/2021.
+//  Copyright © 2021 Nimble Co. Ltd. All rights reserved.
+//
+
+import UIKit
+
+extension UIBarButtonItem {
+
+    func setIdentifier(_ identifier: String) {
```

Comment:
What do you think using a protocol for this? Example
```swift
protocol AccessibilitySettable {
    func setIdentifier(_ identifier: String)
}

extension UIView: AccessibilitySettable {
    func setIdentifier(_ identifier: String) {
        accessibilityIdentifier = identifier
        accessibilityLabel = identifier
    }
}

extension UIBarButtonItem: AccessibilitySettable {
    func setIdentifier(_ identifier: String) {
        accessibilityIdentifier = identifier
        accessibilityLabel = identifier
    }
}

// then our UIViewController extension
extension UIViewController {
    
    func setIdentifiersForViews(with info: [String: AccessibilitySettable?]) {
        info.forEach { (identifier, view) in
            view?.setIdentifier(identifier)
        }
    }
}
```

Reply:
Thanks for your advise. Let me give a try. 👍👍👍